prometheus/conf/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+    monitor: 'desec-slave'
```

Comment:
do we need to have different names for different slave instanes?

Reply:
I don't think so. When you do Prometheus federation (i.e. you have a Prometheus instance on a monitoring server that collects metrics from other Prometheus instances), I think you can set up a metric attribute that identifies the scraped instance.

Reply:
I also considered configuring this here, for clarity. However, we'd have to use an env variable, which Prometheus does not support. As a result, the usual exercise with `envsubst` would be required, which would require us to replace the container's entrypoint script, and down the rabbit hole you go. :-) Let's do that if the other approach turns out unsufficient